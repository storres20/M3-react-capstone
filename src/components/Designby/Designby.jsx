import React from 'react';
import './Designby.scss';

function Designby() {
  return (
    <div className="designContainer">
      <p>
        Designed by
        {' '}
        <a href="https://myport20.netlify.app/" target="_blank" rel="noreferrer"><b>ITALO</b></a>
      </p>
    </div>
  );
}

export default Designby;
