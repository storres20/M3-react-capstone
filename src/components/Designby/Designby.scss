.designContainer {
  color: white;
  height: 50px;
  font-size: 18px;
  display: flex;
  align-items: center;
}

.designContainer > p {
  margin: 0;
}

.designContainer > p > a {
  text-decoration: none;
}

.designContainer > p > a > b {
  color: #0d6efd;
}

.designContainer > p > a > b:hover {
  color: #0343a2;
}
