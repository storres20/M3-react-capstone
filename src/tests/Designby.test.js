import React from 'react';
import renderer from 'react-test-renderer';
import Designby from '../components/Designby/Designby';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Designby />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
