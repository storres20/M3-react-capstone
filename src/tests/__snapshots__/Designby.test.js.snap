// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="designContainer"
>
  <p>
    Designed by
     
    <a
      href="https://myport20.netlify.app/"
      rel="noreferrer"
      target="_blank"
    >
      <b>
        ITALO
      </b>
    </a>
  </p>
</div>
`;
